promxy.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: promxy
+  version: 0.0.90
+  epoch: 0
+  description: An aggregating proxy to enable HA prometheus.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jacksontj/promxy
+      tag: v${{package.version}}
+      expected-commit: 114942b664615bc68a67f56d2e458d7ca88f18d2
+
+  - uses: go/build
+    with:
+      packages: ./cmd/promxy
+      tags: netgo,builtinassets
+      output: promxy
+
+  - uses: go/build
+    with:
+      packages: ./cmd/remote_write_exporter
+      output: remote_write_exporter
+
+subpackages:
+  - name: promxy-compat
+    description: "Compatibility package to allow replacing promxy image"
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}/bin"
+          ln -sf /usr/bin/promxy "${{targets.subpkgdir}}/bin/promxy"
+          ln -sf /usr/bin/remote_write_exporter "${{targets.subpkgdir}}/bin/remote_write_exporter"
+
+update:
+  enabled: true
+  github:
+    identifier: jacksontj/promxy
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        promxy --version
```

Comment:
can we considering adding a help message test for the other binary here? by default it's giving `exit 1` on invoking `--help` but we can do something like this. 

```bash
/ # remote_write_exporter --help | grep drop-stale
      --drop-stale       drop metrics that are written with StaleNaN
/ # echo $?
0
```

Reply:
Good point! I was checking `--help` but did not think to `| grep` it. Done.